core/layouts/boxton/layout--boxton.tpl.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * @file
+ * Template for the Boxton layout.
+ *
+ * Variables:
+ * - $title: The page title, for use in the actual HTML content.
+ * - $messages: Status and error messages. Should be displayed prominently.
+ * - $tabs: Tabs linking to any sub-pages beneath the current page
+ *   (e.g., the view and edit tabs when displaying a node.)
+ * - $action_links: Array of actions local to the page, such as 'Add menu' on
+ *   the menu administration interface.
+ * - $classes: Array of CSS classes to be added to the layout wrapper.
+ * - $attributes: Array of additional HTML attributes to be added to the layout
+ *     wrapper. Flatten using backdrop_attributes().
+ * - $content: An array of content, each item in the array is keyed to one
+ *   region of the layout. This layout supports the following sections:
+ *   - $content['header']
+ *   - $content['top']
+ *   - $content['content']
+ *   - $content['bottom']
+ *   - $content['footer']
+ */
+?>
+<div class="layout--boxton <?php print implode(' ', $classes); ?>"<?php print backdrop_attributes($attributes); ?>>
+  <div id="skip-link">
+    <a href="#main-content" class="element-invisible element-focusable"><?php print t('Skip to main content'); ?></a>
+  </div>
+
+  <?php if ($content['header']): ?>
+    <header class="l-header" role="banner" aria-label="<?php print t('Site header'); ?>">
+      <div class="l-header-inner container container-fluid">
+        <?php print $content['header']; ?>
+      </div><!-- /.container -->
+    </header>
+  <?php endif; ?>
+
+  <div class="l-wrapper">
+    <div class="l-wrapper-inner container container-fluid">
+
+      <?php if ($messages): ?>
+        <div class="l-messages" role="status" aria-label="<?php print t('Status messages'); ?>">
+          <?php print $messages; ?>
+        </div>
+      <?php endif; ?>
+
+      <div class="l-page-title">
+        <a id="main-content"></a>
+        <?php print render($title_prefix); ?>
+        <?php if ($title): ?>
+          <h1 class="page-title"><?php print $title; ?></h1>
+        <?php endif; ?>
+        <?php print render($title_suffix); ?>
+      </div>
+
+      <?php if ($tabs): ?>
+        <nav class="tabs" role="tablist" aria-label="<?php print t('Admin Content Navigation Tabs'); ?>">
```

Comment:
Our standards for strings is sentence case, i.e. `Admin content navigation tabs`.